source/declarations.tex

```diff
@@ -2526,15 +2529,19 @@
 \item
 Otherwise, let $S$ be the entity associated with the scope inhabited by
 the \grammarterm{declarator}.
+\end{itemize}
+Then:
+\begin{itemize}
 \item
 If the \grammarterm{declarator} declares
 an explicit instantiation or a partial or explicit specialization,
 the \grammarterm{declarator} does not bind a name.
```

Comment:
This is actually redundant: such declarations do not introduce names ([temp.explicit]/5, [temp.expl.spec]/4, and [temp.spec.partial.general]/7), so [basic.scope.scope]/2.5 ignores them already.  This is relevant because it is an opportunity to merge into the next sentence (once we're sure exactly what we mean it to mean).